apps/user-profile/package.json

```diff
@@ -78,10 +78,6 @@
     "not ie <= 11",
     "not op_mini all"
   ],
-  "lint-staged": {
```

Comment:
@Enjoy2Live I've deleted it now, is this okay?

Reply:
yessir!